README.md

```diff
@@ -28,6 +28,7 @@ To use SwiftFrame, you need to pass it a configuration file (which is a plain JS
 * `locales`: **optional** a regular expression that can be used to exlude (or include) certain locales during rendering. To only include `fr` and `de` locale for example, use `"fr|de"`. To exclude `ru` and `fr`, use something like `"^(?!ru|fr$)\\w*$"`
 * `deviceData`: an array containing device specific data about screenshot and text coordinates (this way you can frame screenshots for more than one device per config file)
   * `outputSuffix`: a suffix to apply to the output files in addition to the locale identifier and index
+  * `additionalSuffixes`: **optional** an array of additional output suffixes (for example to render the same screenshots a second time for a different device)
```

Comment:
Maybe worth just turning `outputSuffix` itself into an array?

Reply:
And if we do that, the version needs to be 4.0.0 😅

Reply:
That was my initial though as well, but I wanted to avoid bumping to 4.0.0 already 👀 I guess migrating config files shouldn't be too big of a hassle though